CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - BREAKING: Changed createRegistration to now longer return a user URI and instead returns a Registration Object.
 - Changed `convertToDSNPUserId` function to throw correct errors
 - Updated building to Node 16
+- Replaced activity content folder with standalone activity content NPM module
```

Comment:
@dopatraman this should likely be marked as breaking with notes on how to update as we also (correctly) removed the entire `activityContent` export as well as `generators/activityContent`.

Reply:
Will update. Just a heads up though, `generators/activityContent` has not been removed -- it just wraps the NPM package.